config/example-extras.cfg

```diff
@@ -1176,6 +1176,23 @@
 #    Directly sets the default prefix. If present, this value will override
 #    the "default_type".
 
+# The heater_generic section is used to describe a custom/generic heater.
+# Takes the same parameters as any other heater (like a heater_bed),
+# but also takes a gcode_id that is used for temperature reporting
+# and setting the temperature through
+# SET_HEATER_TEMPERATURE ID=<gcode_id> TARGET=<target temp>.
+#[heater_generic my_chamber_heater]
+#gcode_id: C
+#    The ID used for temperature reporting and command to set
+#    the temperature.
+#heater_pin: !PA16
+#sensor_type: NTC 100K MGB18-104F39050L32
+#sensor_pin: PC12
+#control: watermark
+#min_temp: 0
+#max_temp: 120
+#    Typical heater settings used for other types of heaters
```

Comment:
It would be great if the documentation explicitly stated that the fields are documented in the extruder section of the example.cfg file.